neuralmonkey/vocabulary.py

```diff
@@ -363,6 +364,46 @@ def sentences_to_tensor(
         return word_indices, weights
 
 
+    def sentences_to_bare_tensor(
```

Comment:
Again, it should be an argument in a method, not an almost identical copy of it.